packages/website/pages/index.js

```diff
@@ -158,6 +158,16 @@ function About() {
           <a href="https://filecoin.io" className="black">
             Filecoin
           </a>
+          , and are referenced using decentralized, content addressed IPFS URIs
```

Comment:
can we take out the word "decentralized" and add a hyphen to "content-addressed" here (so it just reads "referenced using content-addressed IPFS URIs)...")

Reply:
done 👍